config/parameters.yaml

```diff
@@ -0,0 +1,117 @@
+# Put parameters here that don't need to change on each machine where the app is deployed
+# https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
+parameters:
+    app.most_important_skills_limit: 6 # used for separing data in the planning
+    app.most_important_skills_to_display: [ch_vpsp, ch_vl] # used for displaying a badge near the user name
+    app.slot_interval: '%env(APP_SLOT_INTERVAL)%'
+    app.available_skills:
+        # Caution: the order matters for the planning display
+        ci_bspp:
+            label: CI Réseau BSPP
+            includes: [ci_samu]
+        ci_samu:
+            label: CI Réseau Samu
+            includes: [ci]
+        ch_vpsp:
+            label: Chauffeur VPSP
+            includes: [pse2, ch_vl]
+        ci:
+            label: CI
+            includes: [pse2]
+        pse2:
+            label: PSE2
+            includes: [pse1]
+        pse1:
+            label: PSE1
+            includes: [psc1]
+        ch_vl:
+            label: Chauffeur VL
+            includes: [benevole]
+        infirmier:
+            label: Infirmier.e local.e
+            includes: [benevole]
+        ce_maraude:
+            label: Chef.fe d'équipe maraudes
+            includes: [solidar]
+        solidar:
+            label: Solidar
+            includes: [maraudeur]
+        maraudeur:
+            label: Maraudeur.se
+            includes: [benevole]
+        tsa_coreg:
+            label: TSA / Coreg
+            includes: [pse2]
+        logisticien:
+            label: Logisticien.ne Croix-Rouge
+            includes: [benevole]
+        resp_perm_soc:
+            label: Responsable Permanence Sociale
+            includes: [benevole]
+        anim_fle:
+            label: Animateur.rice de cours de FLE
+            includes: [benevole]
+        psc1:
+            label: PSC1
+            includes: [benevole]
+        benevole:
+            label: Bénévole
+    app.user_properties:
+        - key: vulnerable
+          type: boolean
+          label: Je fais partie des personnes vulnérables
+          columnLabel: Vulnérable
+          help: |
+              - malade chronique<br>
+              - obésité morbide<br>
+              - syndrome grippal<br>
+              - immunodéprimé<br>
+              - personne mineure ou personne de plus de 70 ans<br>
+              - avis défavorable de votre unité locale ou du pole santé (local ou territorial)
+          required: true
+          hidden: false
+          container_class: col-xl-6
```

Comment:
this value can be hard to configure if we allow the app admin to manage the user properties from a web page

Reply:
this is indeed a very low level option for an admin, but at least something _can_ be done for formatting (otherwise, the form looks a bit strange)
![Capture du 2020-05-14 15-08-44](https://user-images.githubusercontent.com/10139766/81938172-e1bc6300-95f4-11ea-8fd8-f9afbc8f68d4.png)

i'm open if you got a clever idea :)
or maybe this won't be needed once the design will be created!

Reply:
let's keep your container_class, but can you add some help on the form, with a default value for example ?